mailbox/store/src/test/resources/eml/mail5.eml

```diff
@@ -0,0 +1,52 @@
+Return-Path: <mailet-api-return-812-benwa=minet.net@james.apache.org>
+Received: from mx2.minet.net (mx2.minet.net [192.168.102.26])
+	 by imap (Cyrus v2.4.16-Debian-2.4.16-4+deb7u1) with LMTPA;
+	 Fri, 15 May 2015 08:36:03 +0200
+X-Sieve: CMU Sieve 2.4
+Received: from localhost (spam.minet.net [192.168.102.97])
+	by mx2.minet.net (Postfix) with ESMTP id 5AC87A19A4F
+	for <benwa@minet.net>; Fri, 15 May 2015 08:36:03 +0200 (CEST)

```

Comment:
anonymization?

Reply:
Benwa is already everywhere (almost all .eml). It's a nickname not a real name. Benoit tell me it's does care.
If you think it is important create a ticket for that I won't do that in that PR